content/en/docs/concepts/containers/images.md

```diff
@@ -237,6 +253,69 @@ Credentials can be provided in several ways:
 
 These options are explained in more detail below.
 
+### Using Azure Container Registry (ACR)
```

Comment:
Please see https://kubernetes.io/docs/contribute/style/content-guide/#what-s-allowed

This detail belongs in Microsoft's documentation, not ours.